### DISCUSSION TITLE
Add blutopia.xyz as provider

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#8119] [Ideas & Feature requests]
Would it be possible to add Blutopia as provider?
https://blutopia.xyz

Can provide temporary access if needed.

**Comment by p0psicles** [comment]
I dunno. I don't give much about torrent providers. What's special about this One? Also no cloudflare?

**Comment by Rouzax** [comment]
This one is geared towards high quality, bitrate and season packs. Similar to Beyond-HD for instance.
